apps/astarte_appengine_api/test/support_v2/generators/datetime.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Astarte.Test.Generators.DateTime do
+  use ExUnitProperties
+
+  @min_default 0
+  @max_default 2_556_143_999
+
+  def date_time!(opts \\ []) do
+    opts = Keyword.validate!(opts, min: @min_default, max: @max_default)
+    [min: min, max: max] = [min: Keyword.fetch!(opts, :min), max: Keyword.fetch!(opts, :max)]
+    date_time!(min, max)
+  end
+
+  def invalid_datetime() do
```

Comment:
generator must produce _valid_ entities. If you want to create a mistake, use a pattern similar to `apps/astarte_appengine_api/test/astarte_appengine_api/v2_group_test.exs`

Reply:
(If you need to reuse an invalid generator in more than one test, you could implement at the beginning of the test file)